commercetools-convenience/src/main/java/io/sphere/sdk/queries/QueryExecutionUtils.java

```diff
@@ -2,11 +2,15 @@
 
 import io.sphere.sdk.client.SphereClient;
 
+import javax.annotation.Nonnull;
 import java.util.List;
 import java.util.concurrent.CompletionStage;
+import java.util.function.Consumer;
+import java.util.function.Function;
 
 /**
  * Provides facilities to fetch all elements matching a query predicate.
+ *
  * @since 1.0.0-RC1
  */
 public final class QueryExecutionUtils {
```

Comment:
Since this class is public, we have to keep the previous method signatures. So we should add new methods for your extension.

Reply:
reverted legacy API: 16d5ea6